src/main/java/edu/kis/powp/jobs2d/command/SinusFactory.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class SinusFactory {
+
+    public static ComplexCommand getSinus(DriverManager driverManager) {
```

Comment:
DriverCommand return type would be sufficient

Passing a manager instead of a driver increases dependencies.